content/docs/user-guide/data-management/index.md

```diff
@@ -0,0 +1,91 @@
+# Data Management with DVC
+
+DVC helps you manage and share arbitrarily large files, datasets, and ML models
+anywhere: mounted drives, network resources (e.g. NAS), external devices, or
+remotely (SSH servers, cloud storage, etc.). Once the project is configured, you
+can manipulate files normally in your local workspace. DVC tracks, restores, and
+synchronizes them across locations.
+
+![]() _Local, external, and remote storage locations_
```

Comment:
Basic idea for figure that may help here (needs simplifying):

![IMG_20221014_154910](https://user-images.githubusercontent.com/1477535/196352047-11a62228-bece-4972-a210-5bf892e24101.jpg)

Reply:
I added some very basic figure drafts @shcheklein, PTAL. Not sure we want 3 or just 2, maybe even just 1. I can improve them somewhat (enough to be temporarily published) after getting some feedback about the page in general. We'd get a designer to work on them later (non-blocking I hope).

Reply:
p.s. I thought about the before/after-style figure but I think it would be more illustrative in docs about data/pipeline versioning: going from a mess with multiple versions present in a project tree to a consistent structure where versions are moved into the time dimension (Git commits).